ts/Dashboards/EditMode/Toolbar/CellEditToolbar.ts

```diff
@@ -87,36 +88,39 @@ class CellEditToolbar extends EditToolbar {
                         (this.menu.parent as CellEditToolbar).onCellOptions();
                     }
                 }
-            },
-            {
-                id: 'destroy',
-                type: 'icon',
-                className: EditGlobals.classNames.menuDestroy,
-                icon: iconURLPrefix + 'destroy.svg',
-                events: {
-                    click: function (this: MenuItem, e: any): void {
-                        const parentNode = this.menu.parent as CellEditToolbar,
-                            editMode = this.menu.parent.editMode,
-                            popup = editMode.confirmationPopup;
-
-                        popup.show({
-                            confirmButton: {
-                                value: editMode.lang.confirmButton,
-                                callback: parentNode.onCellDestroy,
-                                context: parentNode
-                            },
-                            cancelButton: {
-                                value: editMode.lang.cancelButton,
-                                callback: (): void => {
-                                    popup.closePopup();
-                                }
-                            },
-                            text: editMode.lang.confirmDestroyCell
-                        });
-                    }
+            });
+        }
+
+        items.push({
```

Comment:
At this point, we can implement in a similar fashion the `destroy` button 😉 
@oysteinmoseng do you think it makes sense?